### PR TITLE
Upgrading terraform code to support v0.12.3

### PR DESCRIPTION
**Pull request by balaguduru**
Upgrading terraform code to support v0.12.3.

### CODE REVIEW

**Review by chrislovecnm — APPROVED** [review]
/approve
/lgtm

File: gke-to-gke-vpn/terraform/main.tf [hunk 1]
```diff
@@ -21,146 +21,153 @@ Creating Clusters and their Node Pools
...
-      tags = ["kc-node"]
-    }
+  name     = "cluster1-nodes"
+  location = var.cluster1-location
+  project  = var.project
+  cluster  = google_container_cluster.cluster-deployment-cluster1.name
+
+  node_config {
+    oauth_scopes = [
+      "https://www.googleapis.com/auth/compute",
```
Comment (bgeesaman) [inline gke-to-gke-vpn/terraform/main.tf:53]: We have to remove the `compute` scope wherever we see it.  Let's remove it here.

File: gke-to-gke-vpn/terraform/main.tf [hunk 2]
```diff
@@ -21,146 +21,153 @@ Creating Clusters and their Node Pools
...
-      tags = ["kc-node"]
-    }
+  name     = "cluster2-nodes"
+  location = var.cluster2-location
+  project  = var.project
+  cluster  = google_container_cluster.cluster-deployment-cluster2.name
+
+  node_config {
+    oauth_scopes = [
+      "https://www.googleapis.com/auth/compute",
```
Comment (bgeesaman) [inline gke-to-gke-vpn/terraform/main.tf:90]: Here too.

File: gke-to-gke-vpn/terraform/main.tf [hunk 3]
```diff
@@ -21,146 +21,153 @@ Creating Clusters and their Node Pools
...
-      tags = ["kc-node"]
-    }
+  name     = "cluster3-nodes"
+  location = var.cluster3-location
+  project  = var.project
+  cluster  = google_container_cluster.cluster-deployment-cluster3.name
+
+  node_config {
+    oauth_scopes = [
+      "https://www.googleapis.com/auth/compute",
```
Comment (bgeesaman) [inline gke-to-gke-vpn/terraform/main.tf:127]: And wherever you see it, really.  :-)

File: gke-to-gke-vpn/terraform/main.tf [hunk 4]
```diff
@@ -21,146 +21,153 @@ Creating Clusters and their Node Pools
...
-    }
-}
+  name     = "cluster4-nodes"
+  location = var.cluster4-location
+  project  = var.project
+  cluster  = google_container_cluster.cluster-deployment-cluster4.name
+
+  node_config {
+    oauth_scopes = [
+      "https://www.googleapis.com/auth/compute",
```
Comment (bgeesaman) [inline gke-to-gke-vpn/terraform/main.tf:164]: 4th and final

File: gke-to-gke-vpn/terraform/main.tf (hunk 1 above)
Comment (bgeesaman) [reply gke-to-gke-vpn/terraform/main.tf:53]:
```
    oauth_scopes = [
      "https://www.googleapis.com/auth/devstorage.read_only",
      "https://www.googleapis.com/auth/logging.write",
      "https://www.googleapis.com/auth/monitoring",
      "https://www.googleapis.com/auth/servicecontrol",
      "https://www.googleapis.com/auth/service.management.readonly",
      "https://www.googleapis.com/auth/trace.append",
    ]
```
is the newest/best defaults now.  Please apply to all 4.  Thanks!

**Review by bgeesaman — COMMENTED** [review]
Scope tweaks, but ready after that.  Thanks!

**Pull request merged**